openlibrary/templates/account/topmenu.html

```diff
@@ -0,0 +1,25 @@
+$def with (mb)
+
+$ readinglog = mb.key in ['currently-reading', 'already-read', 'want-to-read']
```

Comment:
This would be better in the MyBooks class; some `is_readinglog(key: str) -> bool`